packages/usb/usb.1.3.1/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+name: "usb"
+version: "1.3.1"
+authors: "Jeremie Dimino <jeremie@dimino.org>"
+maintainer: "letoh"
+license: "BSD-3-Clause"
+homepage: "https://github.com/letoh/ocaml-usb"
+bug-reports: "https://github.com/letoh/ocaml-usb/issues"
+dev-repo: "git+https://github.com/letoh/ocaml-usb.git"
+
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "ocamlfind"
+  "oasis" {>= "0.3.0"}
+  ("lwt" {>= "2.4.7"} & "lwt" {< "4.0.0"} | "lwt_ppx")
+  "ocamlbuild" {build}
+  "conf-pkg-config" {build}
+]
+
+depexts: [
+  ["libusb-1.0-0-dev"] {os-family = "debian"}
+  ["libusb"] {os-family = "arch"}
+  ["libusb-dev"] {os-family = "alpine"}
+  ["libusb1-devel"] {os-family = "rhel"}
+]
+
+patches: [
+  "patches/use-lwt_ppx.patch" {lwt:version >= "4.0.0"}
+]
+
+build: [
+  ["./configure"]
+  ["./configure" "--enable-tests"] {with-test}
+  [make]
+  [make "doc"] {with-doc}
+  [make "test"] {with-test}
+]
+
+remove: [["ocamlfind" "remove" "usb"]]
```

Comment:
This is not necessary with opam2 (and with it you can also remove the flags line)